pkg/genericapiserver/genericapiserver.go

```diff
@@ -583,10 +583,24 @@ func verifyEtcdServersList(options *options.ServerRunOptions) {
 	}
 }
 
+func verifySecurePort(options *options.ServerRunOptions) {
+	if options.SecurePort < 0 || options.SecurePort > 65535 {
```

Comment:
After reviewing the kubernetes:master code and testing in my env, I confirm that the current implemention is : 0 for turning off secure port, not insecure port. And the kube-apiserver help info also shows: `--secure-port=6443: The port on which to serve HTTPS with authentication and authorization. If 0, don't serve HTTPS at all.`
I modify the condition and message as this. PTAL, @lavalamp , thanks!